### ISSUE TITLE
DevTools Error: Unable to load the page: net::ERR_CONNECTION_TIMED_OUT

### ISSUE THREAD

**Issue by stenod** [GoogleChrome/lighthouse#2686]
**Initial URL**: https://platform.m2g-portal.de/
**Chrome Version**: 61.0.3156.0
**Error Message**: Unable to load the page: net::ERR_CONNECTION_TIMED_OUT
**Stack Trace**:
```
Error: Unable to load the page: net::ERR_CONNECTION_TIMED_OUT
    at Function.assertPageLoaded (https://chrome-devtools-frontend.appspot.com/serve_file/@7a4241c698a71a5cc10ec6135060f4d99af2b96c/audits2_worker/audits2_worker_module.js:640:332)
    at pass.then._ (https://chrome-devtools-frontend.appspot.com/serve_file/@7a4241c698a71a5cc10ec6135060f4d99af2b96c/audits2_worker/audits2_worker_module.js:644:224)
    at <anonymous>
```

**Comment by wardpeet** [comment]
Thanks for sharing @stenod! Can you reproduce this error on the extension or the cli?

**Comment by wardpeet** [comment]
Fixed by #2461 
issue: #2459
